seqr/views/apis/variant_search_api.py

```diff
@@ -501,11 +504,11 @@ def delete_saved_search_handler(request, saved_search_guid):
     return create_json_response({'savedSearchesByGuid': {saved_search_guid: None}})
 
 
-def _check_results_permission(results_model, user):
+def _check_results_permission(results_model, user, session=None):
     families = results_model.families.prefetch_related('project').all()
     projects = {family.project for family in families}
     for project in projects:
-        check_project_permissions(project, user)
+        check_project_permissions(project, user, session)
```

Comment:
Codacy found an issue: [Too many positional arguments for function call](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6211929)